src/linker/ref/Linker/MessageCategory.cs

```diff
@@ -6,9 +6,9 @@ namespace Mono.Linker
 {
 	public enum MessageCategory
 	{
-		Error = 0,
-		Warning,
-		Info,
-		Diagnostic
+		Error = 1,
```

Comment:
Any reason to make these values a bit-field?
Note that technically we've shipped this, so this is a breaking change. I don't mind that much, but we should have a reason - I don't see one right now.